test/e2e/da_posting_integration_test.go

```diff
@@ -0,0 +1,333 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	tastoradocker "github.com/celestiaorg/tastora/framework/docker"
+	"github.com/celestiaorg/tastora/framework/docker/container"
+	tastoracosmos "github.com/celestiaorg/tastora/framework/docker/cosmos"
+	tastorada "github.com/celestiaorg/tastora/framework/docker/dataavailability"
+	"github.com/celestiaorg/tastora/framework/docker/evstack"
+	"github.com/celestiaorg/tastora/framework/testutil/query"
+	tastoratypes "github.com/celestiaorg/tastora/framework/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/bank"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	"github.com/cosmos/ibc-go/v8/modules/apps/transfer"
+	coreda "github.com/evstack/ev-node/core/da"
+	"github.com/evstack/ev-node/da/jsonrpc"
+	seqcommon "github.com/evstack/ev-node/sequencers/common"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/require"
+)
+
+// TestEvNode_PostsToDA spins up celestia-app, a celestia bridge node and an
+// EV Node (aggregator) via tastora, then verifies the EV Node actually posts
+// data to DA by confirming blobs exist in the ev-data namespace via the DA
+// JSON-RPC client.
+func TestEvNode_PostsToDA(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip integration in short mode")
+	}
+
+	configurePrefixOnce.Do(configureCelestiaBech32Prefix)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	defer cancel()
+
+	uniqueTestName := fmt.Sprintf("%s-%d", t.Name(), time.Now().UnixNano())
+
+	dockerClient, networkID := tastoradocker.Setup(t)
+	t.Cleanup(tastoradocker.Cleanup(t, dockerClient))
+
+	encCfg := testutil.MakeTestEncodingConfig(auth.AppModuleBasic{}, bank.AppModuleBasic{}, transfer.AppModuleBasic{}, gov.AppModuleBasic{})
+
+	// 1) Start celestia-app chain
+	chainImage := container.Image{
+		Repository: "ghcr.io/celestiaorg/celestia-app",
+		Version:    "v5.0.10",
+		UIDGID:     "10001:10001",
+	}
+
+	chainBuilder := tastoracosmos.NewChainBuilderWithTestName(t, uniqueTestName).
```

Comment:
side note: a lot of this config is being moved into a helper to reduce the boiler plate in tests like this, I'll update it after it's merged.